core/class_db.cpp

```diff
@@ -359,6 +358,8 @@ uint64_t ClassDB::get_api_hash(APIType p_api) {
 	OBJTYPE_RLOCK;
 #ifdef DEBUG_METHODS_ENABLED
 
+#include "core/version.h"
```

Comment:
I'm not sure about this change, I prefer to have all includes at the top. If needed, you can add a trailing comment specifying what the include is used for.

I haven't seen many C or C++ codebases using such "include right before the use" logic, and while it's pretty common in Python, it also leads to including things several times in different scopes.

Reply:
It's alright I will check for debug_method_enabled in the head of the file again